src/cpp/savefile.cpp

```diff
@@ -264,7 +264,7 @@ std::string get_changed_values(const Ports& ports, void* runtime,
 
             auto refix_old_end = [&base](const Port* _p, char* _old_end)
             { // TODO: remove base capture
-                bundle_foreach(*_p, _p->name, _old_end, NULL,
+                bundle_foreach(*_p, _p->name, _old_end, NULL, 0,
```

Comment:
Isn't this going to trigger an assert (the `0,` is setting the buffersize to zero and any zero size buffer would trigger an assert, no?)

Reply:
That was indeed a problem (which I did not see because I forgot to run the tests, and the CI is not in place *yet*). Previously, in this context, the function did not even access the buffer and the buffer size, but it now does, for the size computations, so I fixed this function call.